### PR TITLE
Torch 2.6 Version Bump

### PR DESCRIPTION
**Pull request by abaheti95**
Ignored pyright issues and modified workflow image

### CODE REVIEW

File: compose_rl/ppo/callback.py [hunk 1]
```diff
@@ -186,7 +186,10 @@ def env_generate(
                     device=cur_device,
                     dtype=prompt_dtype,
                 ) * eos_token_id
-                sequences = torch.cat([sequences, extra_padding], dim=-1)
+                sequences = torch.cat(
+                    [sequences, extra_padding],  # type: ignore
+                    dim=-1,  # type: ignore
```
Comment (bcui-db) [inline compose_rl/ppo/callback.py:191]: Is the `dim` not expected to not work? All good if not just curious

Comment (abaheti95) [reply compose_rl/ppo/callback.py:191]: I was also shocked at this error... but getting this error
```
error: Argument of type "Literal[-1]" cannot be assigned to parameter "dim" of type "str | ellipsis | None" in function "cat"
    Type "Literal[-1]" cannot be assigned to type "str | ellipsis | None"
      "Literal[-1]" is incompatible with "str"
      "Literal[-1]" is incompatible with "ellipsis"
```

**Review by bcui-db — APPROVED** [review]
Awesome thank you for this! LGTM, just one question on the `dim` type ignores

File: compose_rl/ppo/callback.py (hunk 1 above)
Comment (bcui-db) [reply compose_rl/ppo/callback.py:191]: yeah ack, okay this is weird. I think it's fine
